UI/cypress/e2e/cookies.feature

```diff
@@ -70,5 +70,11 @@ Scenario: Allow users to opt out once they have reached the cookies page for fir
     And a Success Banner is displayed
     And user session is not tracked
 
+Scenario: Error banner appears if no option is selected
+    Given the 'view cookies' page is displayed
```

Comment:
There's some inconsistencies in this file in the use of single quotes. Please can you change all the unicode ‘ and ’ characters to the ASCII ' character

Reply:
Done